senza/cli.py

```diff
@@ -25,7 +25,7 @@
                         fatal_error, info, ok)
 from clickclick.console import print_table
 
-import spotinst
+from spotinst.components import elastigroup_api
```

Comment:
This should be `from .spotinst.components import elastigroup_api` since it's a relative import.

Reply:
This won't work because the spotinst package is outside the senza package. Let's merge this one as a absolute import, and move the spotinst package inside senza in the next PR (there are a few more functionalities to be changed)